packages/nimble/R/genCpp_sizeProcessing.R

```diff
@@ -2560,11 +2560,13 @@ sizeReturn <- function(code, symTab, typeEnv) {
     asserts <- recurseSetSizes(code, symTab, typeEnv)
     if(inherits(code$args[[1]], 'exprClass')) {
         if(!code$args[[1]]$isName) {
-##            if(code$args[[1]]$toEigenize == 'yes') {
-            if(anyNonScalar(code$args[[1]])) {
+            liftArg <- FALSE
+            if(code$args[[1]]$toEigenize == 'yes')
```

Comment:
Do I understand correctly that if `code$args[[1]]$toEigenize == 'maybe'`, then the code will not be eigenized? Specifically, if a child node of the code has `toEigenize == 'yes'`, then we still handle this correctly (either by propagating `'yes'` up before we get here, or by deciding not to eigenize)?